### PR TITLE
Update configure networking

### PR DESCRIPTION
**Pull request by lxpollitt**

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit a329aedbd7936b3b471aa37e33c046dd59571c92

https://deploy-preview-3805--calico.netlify.app

File: networking/bgp.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 
 ### Big picture
 
-Configure border gateway protocol (BGP) peering with network infrastructure to distribute routing information.
+Configure BGP (Border Gateway Protocol) between Calico nodes or peering with network infrastructure to distribute routing information.
 
 ### Value
 
-{{site.prodname}} nodes can exchange routing information over BGP to enable reachability for {{site.prodname}} networked workloads without the need for encapsulation.
+{{site.prodname}} nodes can exchange routing information over BGP to enable reachability for {{site.prodname}} networked workloads (Kubernetes pods or OpenStack VMs). In an on-prem deployment this allows you to make your workloads first class citizen across the rest of your network. In public cloud deployments it provides an efficient way of distributing routing information within your Cluster, often used in conjunction with IPIP overlay or cross-subnet modes.
```
Comment (bmckercher123) [inline networking/bgp.md:13]: In an on-premises deployment this allows you to make your workloads first-class citizen across the rest of your network. In public cloud deployments, it provides an efficient way of distributing routing information within your cluster, and is often used in conjunction with IPIP overlay or cross-subnet modes.

File: networking/bgp.md [hunk 2]
```diff
@@ -26,23 +26,28 @@ This how-to guide uses the following {{site.prodname}} features:
 
 #### BGP
 
-**BGP** is a standard protocol for exchanging routing information between two routers in a network. Each router running BGP has one or more **BGP peers** - other routers which they are communicating with over BGP. You can configure {{site.prodname}} nodes to peer with each other, with route reflectors, and with top-of-rack (ToR) routers.
+**BGP** is a standard protocol for exchanging routing information between routers in a network. Each router running BGP has one or more **BGP peers** - other routers which they are communicating with over BGP. You can think of {{site.prodname}}  networking as providing a virtual router on each of your nodes. You can configure {{site.prodname}} nodes to peer with each other, with route reflectors, or with top-of-rack (ToR) routers.
```
Comment (bmckercher123) [inline networking/bgp.md:29]: extra space here...
You can think of {{site.prodname}}^networking

File: networking/bgp.md [hunk 3]
```diff
@@ -26,23 +26,28 @@ This how-to guide uses the following {{site.prodname}} features:
...
 
 #### Common BGP topologies
 
 There are many ways to configure a BGP network depending on your environment. Here are some common ways it is done with {{site.prodname}}.
 
-#### Topologies for public cloud
+#### Full-mesh
 
-{{site.prodname}}’s default behavior is to create a **full-mesh** of internal BGP (iBGP) connections where each node peers with each other. This design works great for small and medium-size deployments (< 100 nodes) in public cloud, but you can hit performance bottlenecks at around 100 nodes.
+When BGP is enabled, {{site.prodname}}’s default behavior is to create a **full-mesh** of internal BGP (iBGP) connections where each node peers with each other. This allows {{site.prodname}} to operate over any L2 network, whether public cloud or private cloud, or, if IPIP is [configured]({{site.baseurl}}/networking/vxlan-ipip) to operate as an overlay over any network that does not block IPIP traffic. {{site.prodname}} does not use BGP for VXLAN overlays.
```
Comment (bmckercher123) [inline networking/bgp.md:37]: the comma after [configured] is critical -- or sentence is confusing.

...or, if IPIP is [configured]({{site.baseurl}}/networking/vxlan-ipip), to operate as an overlay over any network that does not block IPIP traffic. {{site.prodname}} does not use BGP for VXLAN overlays.

File: networking/bgp.md (hunk 2 above)
Comment (caseydavenport) [inline networking/bgp.md:29]: >  think of {{site.prodname}}  networking 

nit, but two spaces here before "networking"

File: networking/bgp.md [hunk 4]
```diff
@@ -26,23 +26,28 @@ This how-to guide uses the following {{site.prodname}} features:
...
+>**Note**: Most public clouds support IPIP. The notable exception is Azure, which blocks IPIP traffic. So if you want to run Calico as an overlay network in Azure, you must [configure {{site.prodname}} to use VXLAN]({{site.baseurl}}/networking/vxlan-ipip).
+{: .alert .alert-info}
+
+Full-mesh works great for small and medium-size deployments of say 100 nodes or less, but at significantly larger scales full-mesh becomes less efficient, and we recommend using route reflectors.
+
+#### Route reflectors
 
-You can also run {{site.prodname}} on public cloud without BGP or route reflectors using {{site.prodname}}’s **VXLAN cross subnet capabilities**. For more information, see [Configure overlay networking]({{ site.baseurl }}/networking/vxlan-ipip).
+To build large clusters of internal BGP (iBGP), **BGP route reflectors** can be used to reduce the number of BGP peerings used on each node. In this model, some nodes act as route reflectors and are configured to establish a full mesh amongst themselves. Other nodes are then configured to peer with a subset of those route reflectors (typically 2 for redundancy), reducing the total number BGP peering connections compared to full-mesh.
```
Comment (caseydavenport) [inline networking/bgp.md:46]: Do we not want to mention that VXLAN works without RRs in public cloud?

**Pull request merged**